uber/templates/fields/attendee.html

```diff
@@ -360,7 +362,7 @@ <h2 class="text-center">Dealer Application</h2>
     <div class="col-sm-6 {% if not attendee.is_new %}form-control-static">
       {{ attendee.badge }}
       &nbsp;&nbsp; {% if not read_only %}[<a href="change_badge?id={{ attendee.id }}">Change</a>]{% endif %}
-    {% else %}>
+    {% else %}">
```

Comment:
Funny what a single missing quotation mark can do. I don't know when this bug was introduced, but admins creating new attendees haven't been able to set their badge type since.